Automattic/jetpack
# Add/substack to social logo

This PR adds the substack logo to the social logo package. 


## Proposed changes:
*  Add the substack logo. This is need to implement p9Jlb4-cwn-p2

### Other information:




## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Start up the storybook locally. ( navigate to `cd jetpack/projects/js-packages/storybook ` and run ` pnpm run storybook:dev` Notice that the new substack icons looks as expected. When you see it in the storybook.
